glib/src/subclass/signal.rs

```diff
@@ -68,9 +68,103 @@ impl fmt::Debug for SignalInvocationHint {
     }
 }
 
+/// In-depth information of a specific signal
+#[repr(transparent)]
+pub struct SignalQuery(gobject_ffi::GSignalQuery);
+
+impl SignalQuery {
+    pub fn signal_name(&self) -> crate::GString {
```

Comment:
```suggestion
    pub fn signal_name<'a>(&self) -> &'a str {
```

(also below instead of `'static`)

Reply:
I do wonder if the signal name required, because g_signal_name is output is annotated as nullable, but no idea about this one :)

Reply:
never mind, it should never be nullable as the SignalId is now a wrapper around NonZeroU32.

Reply:
Yes this only ever returns `NULL` if the signal was invalid, and we catch that earlier :)